### PR TITLE
Add cards from recent wave 3 articles

### PR DESCRIPTION
**Pull request by FreakyDug**
New things:
- non-limited restriction
- Equipped slot restrictions
- Probe droid device

### CODE REVIEW

File: data/upgrades/crew.json [hunk 1]
```diff
@@ -1272,5 +1272,108 @@
...
+      {
+        "factions": ["Galactic Republic", "Separatist Alliance"]
+      }
+    ],
+    "hyperspace": false
+  },
+  {
+    "name": "Count Dooku",
+    "limited": 1,
+    "xws": "countdooku-crew",
```
Comment (guidokessels) [inline data/upgrades/crew.json]: We haven't included the upgrade type suffix if it's the first upgrade with this name

File: data/upgrades/device.json [hunk 2]
```diff
@@ -118,5 +118,46 @@
...
+        "title": "DRK-1 Probe Droids",
+        "type": "Device",
+        "ability": "During the End Phase, you may spend 1 [Charge] to drop or launch 1 DRK-1 probe droid using a speed 3 template. This card's [Charge] cannopt be recovered.",
+        "image": "https://images-cdn.fantasyflightgames.com/filer_public/8d/db/8ddbbf77-aba5-41d2-8917-6866b43dfada/swz30_probe-droid.png",
+        "slots": ["Device"],
+        "charges": {
+          "value": 2,
+          "recovers": 0
+        },
+        "device": {
```
Comment (guidokessels) [inline data/upgrades/device.json]: This should've failed the build as `device` is not in the schema. Seems like the `sides` items don't have `"additionalProperties": false` set.

File: data/upgrades/crew.json (hunk 1 above)
Comment (FreakyDug) [reply data/upgrades/crew.json]: Oh, my bad, I misunderstood an error from the tests and tried adding that as fix and forgot to remove it when I found the problem (which was that I had accidentally duplicated Dooku).

File: data/upgrades/device.json (hunk 2 above)
Comment (FreakyDug) [reply data/upgrades/device.json]: All of the bombs and mines have `device` so I think we need to add it to the schema (or decide to encode information about the device that's dropped elsewhere).

**Pull request merged**